redis/src/types.rs

```diff
@@ -2562,4 +2561,7 @@ pub struct PushInfo {
     pub data: Vec<Value>,
 }
 
-pub(crate) type PushSender = mpsc::UnboundedSender<PushInfo>;
+#[cfg(feature = "aio")]
+pub(crate) type AsyncPushSender = tokio::sync::mpsc::UnboundedSender<PushInfo>;
+
+pub(crate) type SyncPushSender = std::sync::mpsc::SyncSender<PushInfo>;
```

Comment:
Why have you opted for `SyncSender` instead of `Sender`?

Reply:
Because I misread the documentation, and thought that the fact that the channel is asynchronous means it uses `async` 🤦

Reply:
Random, but I've often thought about making a similar change [here](https://github.com/redis-rs/redis-rs/blob/main/redis/src/aio/multiplexed_connection.rs#L343)

Reply:
yup, I'm not sure why the original writer used a buffered channel there, or what would be the ramifications of changing it.